js/src/sdk/utils/pusher.ts

```diff
@@ -3,19 +3,42 @@ import logger from "../../utils/logger";
 const PUSHER_KEY = process.env.CLIENT_PUSHER_KEY || "ff9f18c208855d77a152";
 const PUSHER_CLUSTER = "mt1";
 
-type PusherClient = any;
+type Channel = {
```

Comment:
The `PusherClient` type is defined as `any`. Consider creating a proper type definition to improve type safety and code maintainability. This will help catch potential errors at compile time rather than runtime.